### ISSUE TITLE
Still a relevant project?

### ISSUE THREAD

**Issue by matmill5** [cscorley/srcmlolol#1]
Chris - is this still a relevant project in 2021?

I'd like to use srcML on some Typescript code, but it's not exactly clear, how to, efficiently, add support for new languages.

I want to avoid reinventing the wheel (if possible)...

**Comment by cscorley** [comment]
No, this was simply a proof-of-concept I've published (and should probably archive so it is clear).  I don't think this should be considered as a production-ready drop-in replacement for srcML.  I am not sure srcML is ever going to be easily extendable in terms of adding new languages, though you may find this project can easily take in any ANTLR4 grammar and convert parsed files into XML.
